src/main/java/gim/model/ExerciseTracker.java

```diff
@@ -17,14 +17,24 @@
 public class ExerciseTracker implements ReadOnlyExerciseTracker {
 
     private final ExerciseList exerciseList;
-    private final ExerciseHashMap exerciseHashMap;
+    private ExerciseHashMap exerciseHashMap;
 
-    {
+    /**
+     * Creates an ExerciseTracker
+     */
+    public ExerciseTracker() {
         exerciseList = new ExerciseList();
         exerciseHashMap = new ExerciseHashMap();
     }
 
-    public ExerciseTracker() {}
+    /**
+     * Creates an ExerciseTracker using a given HashMap {@code ehm}
+     * @param ehm
```

Comment:
I noticed you had '@param ehm ExerciseHashMap' in your SavedExerciseListWindow::setExerciseHashMap method, do you want to standardise a certain convention?

Reply:
Necessary changes made